editor/editor_themes.cpp

```diff
@@ -143,7 +143,7 @@ Ref<Theme> create_editor_theme() {
 	Color light_color_1 = base_color.linear_interpolate(Color(1, 1, 1, 1), contrast);
 	Color light_color_2 = base_color.linear_interpolate(Color(1, 1, 1, 1), contrast * 1.5);
 
-	const int border_width = (border_size % 3) * EDSCALE;
+	const int border_width = MIN(border_size, 3) * EDSCALE;
```

Comment:
Yea modulo also has its advantages... I chnged it, otherwise ppl would be confused when using values like: 7
If u want to we can change it to: `MIN(abs(border_size)), 3)`
I also increased the max bider size from 2*EDSCLAE to 3*EDSCALE. 
Is that fine?